netbox_agent/cli.py

```diff
@@ -5,19 +5,26 @@
 from netbox_agent.vendors.hp import HPHost
 from netbox_agent.vendors.qct import QCTHost
 from netbox_agent.vendors.supermicro import SupermicroHost
+from netbox_agent.vendors.generic import GenericHost
 
 MANUFACTURERS = {
    'Dell Inc.': DellHost,
    'HP': HPHost,
    'HPE': HPHost,
    'Supermicro': SupermicroHost,
    'Quanta Cloud Technology Inc.': QCTHost,
+   'Generic': GenericHost,
+   'Default string': GenericHost
```

Comment:
I'd suggest removing this diff as we'll fall into the exception catching block anyway.